wct-core/src/main/webapp/jsp/target-profile.jsp

```diff
@@ -42,14 +42,12 @@
   }
 
   function toggleProvideOverrides(harvesterTypeValueSelected) {
-    if (harvesterTypeValueSelected == "HERITRIX1") {
-      $('#excludeFiltersRow').show();
-      $('#forceAcceptFiltersRow').show();
-      $('#excludedMimeTypesRow').show();
+    if (harvesterTypeValueSelected == 'HERITRIX1') {
```

Comment:
Should statements like this be matching on the enum instead?

Reply:
This is Javascript